sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RedundantJumpStatement.CSharp9.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+var a = new Action(() =>
```

Comment:
is this related to C#  9 syntax? if not, maybe you can remove it

this functionality is tested at line 13

Reply:
Indeed it is very similar with the one from line 13. I will remove it.